pkg/cmd/extension/command.go

```diff
@@ -220,6 +246,26 @@ func checkValidExtension(rootCmd *cobra.Command, m extensions.ExtensionManager,
 	return nil
 }
 
+func isBinExtension(client *http.Client, repo ghrepo.Interface) (isBin bool, err error) {
+	hs, err := hasScript(client, repo)
+	if err != nil || hs {
+		return
+	}
+
+	_, err = fetchLatestRelease(client, repo)
+	if err != nil {
+		httpErr, ok := err.(api.HTTPError)
+		if ok && httpErr.StatusCode == 404 {
+			err = nil
+			return
+		}
+		return
+	}
+
+	isBin = true
```

Comment:
Does this assume that the presence of any Releases in a repo means that it's a binary extension? Could we detect te presence of at least some assets looking like they are binary extensions and make an assumption based on that? I'm thinking that there might be non-binary extensions that want to utilize their Releases for purposes other than attaching a compiled binary to.